src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -889,13 +889,13 @@ private void BeginExecuteNonQueryInternalReadStage(TaskCompletionSource<object>
             }
         }
 
-        private void VerifyEndExecuteState(Task completionTask, String endMethod)
+        private void VerifyEndExecuteState(Task asyncResult, String endMethod)
```

Comment:
Seems a bit backwards to change the name of the parameter from the more applicable "completionTask" to the less applicable "asyncResult", simply to be able to use ```nameof(asyncResult)``` instead of ```"asyncResult"```.